src/liger_kernel/ops/experimental/gemm_split_k_fp8_e4m3.py

```diff
@@ -0,0 +1,302 @@
+# adapted from: https://github.com/pytorch-labs/applied-ai/blob/main/kernels/triton/inference/fp8/splitk_gemm_fp8.py
```

Comment:
should we adapt the BSD3 license header from the original repo? @ByronHsu

Reply:
we can check with legal internally